tpot/tpot.py

```diff
@@ -288,6 +289,182 @@ def score(self, training_features, training_classes, testing_features, testing_c
 
         return self._evaluate_individual(self.optimized_pipeline_, training_testing_data)[0]
 
+    def export(self, output_file_name):
+        """Exports the current optimized pipeline as Python code.
+
+        Parameters
+        ----------
+        output_file_name: string
+            String containing the path and file name of the desired output file
+
+        Returns
+        -------
+        None
+        """
+        if self.optimized_pipeline_ == None:
```

Comment:
Isn't the equality more explicit?

I just noticed in the `score()` function it now says:

    if self.optimized_pipeline_ is None:

Personally, I think the equality is better since it is explicitly checking for the default state (`None`). I'm open to being convinced otherwise though.